src/Configuration.ts

```diff
@@ -103,6 +104,7 @@ const defaultConfiguration: Configuration = {
 
   healthIntervalInSeconds: 30,
   lowWalletBalanceInBitcoin: 1,
+  feeEstimateBitcoinConfMinTargetBlock: 1,
```

Comment:
Maybe just `feeEstimateMinTargetBlock`? (without the `BitcoinConf`)

Reply:
Sounds good!